epilogue-processor/src/test/java/edu/wpi/first/epilogue/processor/AnnotationProcessorTest.java

```diff
@@ -2348,6 +2348,13 @@ public boolean isAtTarget() {
       import edu.wpi.first.epilogue.logging.LogMetadata;
 
       public class ExampleLogger extends ClassSpecificLogger<Example> {
+        // Cached LogMetadata for element isAtTarget with @DependsOn annotations
+        private static final LogMetadata $metadata_edu_wpi_first_epilogue_Example_isAtTarget;
+
+        static {
+          $metadata_edu_wpi_first_epilogue_Example_isAtTarget = new edu.wpi.first.epilogue.logging.LogMetadata(java.util.List.of("position", "target", "tolerance"));
```

Comment:
Let's import these types. Even though the file is generated, it'd be nice to be more human-friendly